src/app/core/connection.ts

```diff
@@ -280,7 +280,7 @@ export default class Connection {
             // gets teh feature defined by the id
             feature = ConnectionUtils.getFeatureFromID(this._featureIDs[i]);
             console.log(feature);
-            renderedfeature = FeatureRenderer2D.renderFeature(feature);
+            renderedfeature = FeatureRenderer2D.renderFeature(feature, undefined);
```

Comment:
Same as before, either pass `None` or change the method definition to default it to `None`.